server/controllers/images.js

```diff
@@ -0,0 +1,56 @@
+const express = require('express');
+const aws = require('aws-sdk');
+const multer = require('multer');
+const multerS3 = require('multer-s3');
+const router = express.Router();
+const auth = require('../middlewares/auth');
+
+let db;
```

Comment:
Just a thought, if you used a ternary here, you could make `db` a `const`.

Reply:
that's true... also just realised I don't need the other db here, because I can't simulate an image upload from cypress, so testing for that functionality will only happen by client tests.